Feed/index.html

```diff
@@ -48,84 +48,103 @@
                     <div class="card mb-4 d-flex justify-content-center">
                         <div class="card-header">
-                            <h5>What's on your mind today?</h5>
-                        </div>
-                        <div class="card-body">
-                            <textarea class="form-control" rows="3" placeholder="Write your post..."></textarea>
-                        </div>
-                        <div class="card-footer">
-                            <button class="btn btn-success">Create Post</button>
+                            <h5>What's on your mind today, ?</h5>
                         </div>
+                        <form id="newPostForm">
+                            <div class="card-body">
+                                <textarea class="form-control" id="post-form-title" rows="1" placeholder="Title" required></textarea>
+                                <textarea class="form-control" id="post-form-body" rows="3" placeholder="Write your post..."></textarea>
+                                <textarea class="form-control" id="post-form-media" rows="1" placeholder="Media (add URL)"></textarea>
+                            </div>
+                            <div class="card-footer">
+                                <button class="btn btn-success" id="submit-post">Create Post</button>
+                            </div>
+                        </form>
                     </div>
-                    <div class="input-group mb-3 ">
-                        <select class="custom-select" id="inputGroupSelect01">
-                            <option selected>Sort by</option>
-                            <option value="1">Date</option>
-                            <option value="2">A-Z</option>
-                            <option value="3">Newest posts</option>
-                        </select>
+                </div>
+            </div>
+        </div>
+        <div class="d-flex justify-content-center mt-5">
+            <h2 id="post-success" class="text-success" style="display: none; "> Successfully created post</h2>
+        </div>
+        
+        <div class="dropdown">
+            <a class="btn btn-secondary dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Filter
+            </a>
+          
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+            <a class="dropdown-item" id="filter-default" href="#">None</a>
+              <a class="dropdown-item" id="filter-likes" href="#">Likes</a>
+              <a class="dropdown-item" id="filter-comments" href="#">Comments</a>
+            </div>
+          </div>
+
+        
+          <div id="edit-post-modal" class="modal">
+            <div class="modal-content">
+                <span class="close-button">&times;</span>
+                <h2>Edit Post</h2>
+                <form id="edit-post-form">
```

Comment:
This looks good. Consider this and fix the new post form elements

Reply:
New post form has been updated